department-of-veterans-affairs/vets-website
# VACMS-11590 Remove facilities_ppms_suppress_community_care flipper

## Are you removing, renaming or moving a folder in this PR?

### :warning: TeamSites :warning:
Did you change site-wide styles, platform utilities or other infrastructure?

## Summary
Remove unused `facilities_ppms_suppress_community_care` flipper. Includes removing Cypress and unit tests that tested this functionality with the flipper on and off.

- Original PR where this logic was added: https://github.com/department-of-veterans-affairs/vets-website/pull/13638
- Github-wide search for this flag: https://github.com/search?q=org%3Adepartment-of-veterans-affairs+facilities_ppms_suppress_community_care&type=code

Flipper is **OFF** in prod:
<img width="852" alt="Screenshot 2024-11-20 at 4 59 00 PM" src="https://github.com/user-attachments/assets/b91fc76e-cbe6-4d48-89a6-5b03762b31fe">

Flipper is **OFF** in staging:
<img width="848" alt="Screenshot 2024-11-20 at 4 59 06 PM" src="https://github.com/user-attachments/assets/88e6d704-7b14-4bb7-840a-2a98c2303482">

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11590

## Testing done
No functionality changes are expected.

1. Go to `/find-locations`
2. Enter a zip code (I used 78258)
3. Select `Community providers (in VA's network)` for Facility Type
4. When the Service type field is available, type a service type (I used `Dentist`)
5. Validate the results you receive against production
